lib/renaissance_web/helpers/auth.ex

```diff
@@ -1,7 +1,19 @@
 defmodule RenaissanceWeb.Helpers.Auth do
+  alias Renaissance.Users
+
   def signed_in?(conn) do
     current_user_id = Plug.Conn.get_session(conn, :current_user_id)
 
     current_user_id != nil
   end
+
+  def current_user(conn) do
+    case Plug.Conn.get_session(conn, :current_user_id) do
```

Comment:
wouldn't `if` be better?

Reply:
Changed to use pattern matching, thoughts?

Reply:
I could see inlining the nil one, but it looks cleaner IMO

Reply:
Fixed!